Makefile

```diff
@@ -1,9 +1,9 @@
 maint:
```

Comment:
Please do not change this file in this PR - this is unrelated and unnecessary.

Reply:
this makefile refers to python 3.7 : I've updated it to match drop any reference to this release